eng/Versions.props

```diff
@@ -67,6 +67,7 @@
   </PropertyGroup>
   <PropertyGroup Label="Manual">
     <AzureStorageBlobsVersion>12.6.0</AzureStorageBlobsVersion>
+    <AzureStorageQueuesVersion>12.8.0</AzureStorageQueuesVersion>
```

Comment:
I believe this is the latest stable version, but I don't know if we have a process for how we choose which version we want to use.

Reply:
For these particular apis, latest stable should work.

Reply:
Please choose an Azure.Storage.Queues version that matches the dependencies of Azure.Storage.Blobs. There have been binary breaking changes in the past for these packages and I'd rather avoid missing methods and type load failures.